packages/uploadthing/src/internal/handler.ts

```diff
@@ -679,3 +674,7 @@ const handleUploadAction = (opts: {
       fiber,
     };
   }).pipe(Effect.withLogSpan("handleUploadAction"));
+
+const OkHttpClient = HttpClient.HttpClient.pipe(
```

Comment:
For the sake of consistency either remove this abstraction or use if everywhere you do `.pipe(filterOk)`